src/js/midi/helpers.js

```diff
@@ -1,5 +1,10 @@
-const getNameFromPort = (port) => `${port.name} (version ${port.version}) ${port.manufacturer}`
+const getNameFromPort = (port) => {
+  const { name, version, manufacturer } = port
+  return `${name} (version ${version}) ${manufacturer}`
+}
 
-const getAllKeys = () => R.unfold((n) => (n > keyIdMax ? false : [n, n + 1]), keyIdMin)
-const getWhiteKeys = R.compose(R.map(parseInt), R.keys, () => whiteOnlyMap)
-const getBlackKeys = R.converge(R.difference, [getAllKeys, getWhiteKeys])
+const allMidiKeys = [...Array(128).keys()] // [0, 1, 2, ..., 127]
+
+const whiteMidiKeys = Object.keys(whiteOnlyMap).map((id) => parseInt(id))
```

Comment:
Nice. Did you consider re-casing this to `ALL_CAPS` and moving to `constants.js` now that it's a constant?